templates/web/fixmystreet.com/before_wrapper.html

```diff
@@ -0,0 +1,18 @@
+<div class="survey-banner">
+    <p>
+        <span class="survey-banner__text">Share your views and help us improve FixMyStreet for&nbsp;everyone</span>
+        <a class="survey-banner__cta" href="https://www.surveygizmo.com/s3/5506345/46e9932d0fbf" target="_blank">Take our short survey</a>
+    </p>
+    <button class="survey-banner__close">Close</button>
+</div>
+<script nonce="[% csp_nonce %]">
+if ( document.cookie.indexOf('has_closed_march_2020_survey') === -1 && Math.random() > 0 ) {
```

Comment:
presumably not 0 :)
This will somehow need to know not to run in test because its appearance is breaking all the map-click tests, I'm afraid.